### DISCUSSION TITLE
[Native DLL] Fixed size array

### DISCUSSION THREAD

**Discussion by ghost** [pascalabcnet/pascalabcnet#3021] [General]
For example, how to handle fixed size array of integers in structs?

**Comment by SunSerega** [comment]
Basically the same as https://github.com/pascalabcnet/pascalabcnet/discussions/3020

You can also specify array size in `[MarshalAs]`.
Or you can make a fixed-size `record` and use `Marshal.ReadInt32`, etc. to read simple items at offsets. Anything more complex - you can do by dereferencing pointers manually, but it **will** be ugly.
